docs/user-guide/agile-boards/README.md

```diff
@@ -8,11 +8,9 @@ keywords: agile boards, Kanban, Scrum, agile project management, action boards
 
 # Boards for Agile Project Management
 
-Boards support agile project management methodologies, such as Scrum or Kanban.
+Boards support agile project management methodologies, such as Scrum or Kanban. OpenProject agile boards enable you to keep track tasks, bugs, features, risks and any other work package types. Boards consist of lists (columns) and cards (work packages). You can choose between a Basic board and various advanced Action boards.
```

Comment:
@MayaBerd I think Agile should always be written in capital "A" as its a methodology name.